x-pack/plugins/observability_solution/infra/public/components/asset_details/hooks/use_metrics_charts.ts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { findInventoryModel } from '@kbn/metrics-data-access-plugin/common';
+import useAsync from 'react-use/lib/useAsync';
+
+export const useHostFlyoutViewMetricsCharts = ({
```

Comment:
My point here is should we link a UI with a hook that fetches data? Shouldn't this hook be named according to type of the data it fetches?

Like the `useKubernetesSectionMetricsCharts` it could be like it was before `useKubernetesMetricsCharts`, it's clear that this hook will fetch kubernetes metrics, so if I were to build a new page in the future related to kuberneted this hook would be clear what it does.

But like I said maybe I don't get the data structure.

Reply:
I got it.  For instance, `useFlyoutMetricsCharts` as it was before, it's because we show different host metrics in the flyout compared to the page view. Therefore, we had `useFlyoutMetricsCharts` and `useHostMetricsCharts`. 

When changing the name, it made sense to name them according to the UI component that they are consumed, as both hooks will return metrics for hosts. To keep consistency, `useKubernetesMetricsCharts` was renamed to `useKubernetesSectionMetricsCharts`. Does that make sense to you? In case we need to reuse any of those hooks elsewhere, we can always adapt their name.

Reply:
>I got it. For instance, useFlyoutMetricsCharts as it was before, it's because we show different host metrics in the flyout compared to the page view

Lets keep `useHostFlyoutViewMetricsCharts` then.